PatientRegistrationApp/src/components/SinglePatientEmergencyContact.vue

```diff
@@ -20,13 +24,15 @@
       <v-col class="d-flex" cols="5" sm="5">
         <v-text-field
           label="Phone Number"
+          v-model="contactPhoneNumber"
           prepend-icon="mdi-menu-right"
         ></v-text-field>
       </v-col>
+
       <!-- Phone Number Type -->
       <v-col class="d-flex" cols="5" sm="3">
         <v-select
-          v-model="e7"
+          v-model="contactPhoneNumberType"
```

Comment:
Yea naming wise, `contactPhoneNumberType` (mostly the contact bit) is still unclear as to who it refers to. I would just add emergency to it

Reply:
updated accordingly

Reply:
Updated accordingly